tianshou/policy/modelfree/sac.py

```diff
@@ -130,9 +133,6 @@ def forward(  # type: ignore
         if self._noise is not None and not self.updating:
             act += to_torch_as(self._noise(act.shape), act)
```

Comment:
action noise is supposed to be added at collecting state (`self.training`=`True` and `self.updating`=`False`) only but it is added at  testing state (`self.training`=`False` and `self.updating`=`False`) also now.